File: C/mario.c
#include <stdio.h>
#include <cs50.h>

void printBlock(int l, int c);

int main(void)
{
    int l = get_int("lines: ");
    int c = get_int("columns: ");

    printBlock(l,c);
}

void printBlock(int l, int c)
{
    for(int i=0;i<l;i++){
        for(int j=0;j<c;j++){
            printf("#");
        }
        printf("\n");
    }
}

File: Python/mario.py
from cs50 import get_int

def main():
    n = get_int("Height: ")
    draw(n)

def draw(n):
    if n <= 0:
        return

    draw(n - 1)
    print("#" * n)  # no need to use loops. The * operator will repeat the string many times


main()
